src/Objs/Font/FontEditingConfig.ts

```diff
@@ -16,10 +16,10 @@ provideEditingConfig(Font, {
     [
       'blob',
       (blob: Binary | null) => {
-        if (blob?.raw().contentType() === 'font/woff2') return
+        if (blob?.raw().contentType().startsWith('font/')) return
 
         return {
-          message: 'Only WOFF2 font files are supported.',
+          message: 'The uploaded file is not a font file.',
```

Comment:
* [ ] FYI thanks to `ObjClassForContentType` and the Content Browser config it's extremely unlikely that this code path is ever hit. It's like validating that an Image blob is an image. I'd remove this check.